steam_wrapper/steam_wrapper.py

```diff
@@ -266,10 +272,14 @@ def main(steam_binary=STEAM_PATH):
     logging.info("https://github.com/flathub/com.valvesoftware.Steam/wiki")
     current_info = read_flatpak_info(FLATPAK_INFO)
     check_allowed_to_run(current_info)
-    migrate_config()
-    migrate_data()
+    should_restart = migrate_config()
+    should_restart += migrate_data()
     migrate_cache()
```

Comment:
I thing we should do the same for cache just for symmetry. Also migrating cache in two steps, too, may be a good idea, but this can be considered later.

Reply:
Fixed.